contracts/test/integration/IntegrationTestVoltMigratorRouter.t.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {ERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {ICore} from "../../core/ICore.sol";
+import {IVolt, Volt} from "../../volt/Volt.sol";
+import {VoltMigrator} from "../../volt/VoltMigrator.sol";
+import {OraclePassThrough} from "../../oracle/OraclePassThrough.sol";
+import {MigratorRouter} from "../../pcv/MigratorRouter.sol";
+import {VoltV2} from "../../volt/VoltV2.sol";
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {PriceBoundPSM, PegStabilityModule} from "../../peg/PriceBoundPSM.sol";
+import {IPegStabilityModule} from "../../peg/IPegStabilityModule.sol";
+import {ERC20CompoundPCVDeposit} from "../../pcv/compound/ERC20CompoundPCVDeposit.sol";
+import {TempCoreRef} from "../../refs/TempCoreRef.sol";
+
+import "hardhat/console.sol";
+
+contract IntegrationTestVoltMigratorRouterTest is DSTest {
```

Comment:
could we use vip-13 here instead of doing all this deployment in the setup function?